core/src/com/geekbrains/td/Antiking.java

```diff
@@ -0,0 +1,179 @@
+package com.geekbrains.td;
+
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.math.*;
+
+public class Antiking extends Monster {
+    private float animationTimer, timePerFrame;
+    private BitmapFont font;
+    private int destCellX;
+    private int destCellY;
+    private float speed;
+
+    public Circle getHitArea() {
```

Comment:
First, the method overlaps with `getHitArea()` from Monster.class without @Override. Second, it already exists in the parent class. Doesn't make any sense.